README.md

```diff
@@ -79,6 +79,17 @@ Note:
 
 - Please grep for all `ERROR` from the output log files to get the list of missing and mismatched records.
     - Note that it lists differences by primary-key values.
+- If you would like to redirect such logs into a separate file, you could use the `log4j2.properties` file [provided here](./src/resources/log4j2.properties) as shown below
+
+```
+./spark-submit --properties-file cdm.properties \
+--conf spark.cdm.schema.origin.keyspaceTable="<keyspacename>.<tablename>" \
+--conf "spark.executor.extraJavaOptions='-Dlog4j.configurationFile=log4j2.properties'" \ 
+--conf "spark.driver.extraJavaOptions='-Dlog4j.configurationFile=log4j2.properties'" \ 
```

Comment:
What will be the name of the 2 log files that will get generated? Could we show that as an example? TY

Reply:
Yes, the details of the log file are in the [log4j2.properties](https://github.com/datastax/cassandra-data-migrator/blob/feature/separate-diff-logs/src/resources/log4j2.properties) file. CDM users can change the details as needed, but if they use the file as-is, the logs will be directed 3 ways as below

- All non app related logs (i.e. logs generated by libraries & not CDM code) will continue to be written to console
- All logs generated by CDM code except for the `diff` related logs (i.e. details of `missing` & `mismatched` rows) will be recorded in `./cdm_logs/cdm.log` file. 
- All `diff` related logs (i.e. details of `missing` & `mismatched` rows) will be recorded in `./cdm_logs/cdm_diff.log` file.

**Note**: All log files will be rolled when they cross the set size limit (default is 10MB), there can be 100 rollovers by default.

Reply:
For context, most customers will not use this feature, but some customers who plan to run only validation & expect to have a lot of `diff` rows want a separate file for such records in a separate file instead of having to `grep` & find it.